### ISSUE TITLE
inference time of f-convnet

### ISSUE THREAD

**Issue by WWW2323** [zhixinwang/frustum-convnet#42]
![image](https://user-images.githubusercontent.com/39642019/94449166-61328380-01de-11eb-8a69-05881a8a7cf0.png)
hi，could you tell me how do you measure the inference time of f-convnet at 0.47s on the KITTI benmark，does it consist of 2d detection time、frustum extracting time and f-convnet model time？did you use a batch-size which bigger than 1 to inference all frustums in one image？when i inference  a batch of frustums(batch size: 32),  f-convnet  just consume  0.03s，is there something wrong with my code?

**Comment by zhixinwang** [comment]
@WWW2323  Yes. The inference time includes the 2D RGB detection (most of time). And for F-ConvNet, we measure the time of 32 frustums, including first stage and refinement stage. Note that we assume that one image contains 32 2D bboxes on average. Actually, the number of 2D bboxes of most images  is less than 32 if you have a good 2D detector. The inference time here is conservative, and also has a lot of optimization space.

**Comment by MoreLyxixi** [comment]
请问如何获取测试集的2D检测结果呢
